locales.js

```diff
@@ -28,6 +28,81 @@ const locales = {
       `December`
     ]
   },
+  'zh_zh-CN': {
+    days: [
+      `星期天`,
+      `星期一`,
+      `星期二`,
+      `星期三`,
+      `星期四`,
+      `星期五`,
+      `星期六`
+    ],
+    months: [
+      `一月`,
+      `二月`,
+      `三月`,
+      `四月`,
+      `五月`,
+      `六月`,
+      `七月`,
+      `八月`,
+      `九月`,
+      `十月`,
+      `十一月`,
+      `十二月`
+    ]
+  },
+  'zh_zh-Hans-CN': {
```

Comment:
Cannot include a locale in the file more than once.  `zh` is included in this set and the one prior.  It should only be included in the set that should serve as a fallback for all zh-based locales.